test/fw/bin/pbs_as

```diff
@@ -166,177 +167,175 @@ if __name__ == '__main__':
                 sys.stdout.write(str(rc))
                 sys.stderr.write(repr(e))
                 sys.exit(e.rc)
 
     elif cmd == 'rerunjob':
         if objid is None:
-            print '1'
+            print('1')
         else:
             (s, data) = _load_data(objfile, servername, user)
             try:
                 rc = s.rerunjob(objid, extend=extend)
                 sys.stdout.write(str(rc))
-            except PbsRerunError, e:
+            except PbsRerunError as e:
                 sys.stdout.write(str(rc))
                 sys.stderr.write(repr(e))
                 sys.exit(e.rc)
 
     elif cmd == 'orderjob':
         if objid is None:
-            print '1'
+            print('1')
         else:
             (s, jobid2) = _load_data(objfile, servername, user)
             try:
                 rc = s.orderjob(objid, str(jobid2), extend=extend)
                 sys.stdout.write(str(rc))
-            except PbsOrderError, e:
+            except PbsOrderError as e:
                 sys.stdout.write(str(rc))
                 sys.stderr.write(repr(e))
                 sys.exit(e.rc)
 
     elif cmd == 'runjob':
         if objid is None:
-            print '1'
+            print('1')
         else:
             (s, location) = _load_data(objfile, servername, user)
             try:
                 rc = s.runjob(objid, str(location), extend=extend)
-            except PbsRunError, e:
+            except PbsRunError as e:
                 rc = e.rc
-            print str(rc)
+            print(str(rc))
 
     elif cmd == 'movejob':
         if objid is None:
-            print '1'
+            print('1')
         else:
             (s, destination) = _load_data(objfile, servername, user)
             try:
                 rc = s.movejob(objid, str(destination), extend=extend)
                 sys.stdout.write(rc)
-            except PbsMoveError, e:
+            except PbsMoveError as e:
                 sys.stdout.write(str(rc))
                 sys.stderr.write(repr(e))
                 sys.exit(e.rc)
 
-    """
-    elif cmd == 'alterresv':
-        if objid is None:
-            print '1'
-        else:
-            (s, attrib) = _load_data(objfile, servername, user)
-            try:
-                rc = s.alterresv(objid, attrib, extend=extend)
-                sys.stdout.write(str(rc))
-            except PbsAlterError, e:
-                sys.stdout.write(str(rc))
-                sys.stderr.write(repr(e))
-                sys.exit(e.rc)
-    """
+    # elif cmd == 'alterresv':
+    #    if objid is None:
+    #        print '1'
+    #    else:
+    #        (s, attrib) = _load_data(objfile, servername, user)
+    #        try:
+    #            rc = s.alterresv(objid, attrib, extend=extend)
+    #            sys.stdout.write(str(rc))
+    #        except PbsAlterError, e:
+    #            sys.stdout.write(str(rc))
+    #            sys.stderr.write(repr(e))
+    #            sys.exit(e.rc)
```

Comment:
This is commented in the master branch too. But it is a multiline comment. I am changing it back to how it is originally. As I do not know why it was commented, I don't want to delete these lines.

Reply:
> As I do not know why it was commented, I don't want to delete these lines.

This was added as commented code by #346, and IIRC then it was commented because PTL_API mode is broken currently

Reply:
ignore this - i missed that it was a multiline comment.